### PR TITLE
0.8.3 Changelog

### PR DESCRIPTION
**Pull request by DStatWriter**
Changes to reflect new version 0.8.3

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,12 @@
 
 All notable changes to the COBOL Language Support extension are documented in this file.
 
+## [0.8.3]
+
+- Syntax highlighting for COBOL code enabled.
+- Syntax check for keywords highlights potential errors and mistakes in COBOL code.
+- Autocomplete suggestions  for COBOL keywords while you type
```
Comment (temanbrcom) [inline CHANGELOG.md]: Extra space in this line

Comment (fox0r) [reply CHANGELOG.md]: indeed

Comment (DStatWriter) [reply CHANGELOG.md]: Fixed, thanks

Comment (sergiuilie) [inline CHANGELOG.md]: Please remove the last two points.
About the first point please write something like "Syntax highlighting for variables and paragraphs in COBOL file"
Add the point about java fix

**Pull request merged**